src/transformers/modeling_albert.py

```diff
@@ -600,7 +600,7 @@ def forward(self, hidden_states):
         hidden_states = self.LayerNorm(hidden_states)
         hidden_states = self.decoder(hidden_states)
 
-        prediction_scores = hidden_states
+        prediction_scores = hidden_states + self.bias
```

Comment:
This line was removed because it was a bug. Best to remove it here as well.

Reply:
Fixed